### ISSUE TITLE
.env is not read if the ENVKEY was found in ~/.envkey/apps

### ISSUE THREAD

**Issue by colvin** [envkey/envkey#51]
Local overrides in `.env` within my app are either not read or are not honored if the ENVKEY is found within the `~/.envkey/apps` directory.

```
# Here we see that .env contains a local setting for FOO, and no ENVKEY
$ cat .env
FOO=abcdefg

# Run envkey-source and observe that the output does not match .env
$ envkey-source --json | jq
{
  FOO=gfedcba
}

# Move the ENVKEY into .env
$ cat ~/.envkey/apps/ID.env >> .env

# Here we see that the output now honors the local override in .env
$ envkey-source --json | jq
{
  FOO=abcdefg
}
```

My understanding was that `envkey-source` (and therefore the other SDKs, in our case the Node SDK) would still read `.env` if it is found and would prefer the values in `.env` over its own. This behavior indicates that this is only the case when `.env` contains the ENVKEY.

**Comment by danenania** [comment]
Thanks @colvin, the current functionality is to only look for overrides in the `.env` file where the `ENVKEY` is found (in your case the ~/.envkey/apps dir). This isn't well documented though. I can see the value in applying overrides from a `.env` in the current directory too. We'll take this into account alongside #49

**Comment by colvin** [comment]
Thanks @danenania. We can use subenvironments when there are permanent variances needed, but in some cases we really need the ability to apply ad-hoc overrides. We have a lot of developers in our org using the auto-generated keys in `~/.envkey/apps`, and the complexity involved in being required to move the key to `.env` in order to support local overrides is just too much.

I appreciate you looking into #49, but I've got to be honest this feature is probably way more important than that one because this one seems to violate the "principle of least astonishment", since the behavior is highly contextual, and seems like a regression since we used to be able to override values using `.env` in v1.
